src/lv_fonts/lv_font_template.c

```diff
@@ -0,0 +1,172 @@
+
+#include "lvgl/lvgl.h"
+
+/***********************************************************************************
+ * Copy/Paste the command line instruction
+ **********************************************************************************/
+
+/*-----------------
+ *    BITMAPS
+ *----------------*/
+
+/*Store the image of the glyphs*/
+static const uint8_t gylph_bitmap[] = {
+    /*Unicode: U+0031 (1) */
+    0x38, 0xf8, 0x98, 0x18, 0x18, 0x18, 0x18, 0x18,
+    0x18, 0x18, 0x18, 0xff, 0xff,
+
+    /*Unicode: U+0033 (3) */
+    0x7e, 0x00, 0xff, 0x00, 0x83, 0x80, 0x01, 0x80,
+    0x03, 0x80, 0x3e, 0x00, 0x3f, 0x00, 0x03, 0x80,
+    0x01, 0x80, 0x01, 0x80, 0x83, 0x80, 0xff, 0x00,
+    0x7e, 0x00,
+
+    /*Unicode: U+0041 (A) */
+    0x06, 0x00, 0x06, 0x00, 0x0f, 0x00, 0x0f, 0x00,
+    0x19, 0x80, 0x19, 0x80, 0x19, 0x80, 0x30, 0xc0,
+    0x3f, 0xc0, 0x7f, 0xe0, 0x60, 0x60,  0x60, 0x60,
+    0xc0, 0x30,
+
+    /*Unicode: U+0061 (a) */
+    0x3c, 0x7e, 0x47, 0x03, 0x3f, 0xff, 0xc3, 0xc7,
+    0xff, 0x7b,
+};
+
+
+/*---------------------
+ *  GLYPH DESCRIPTION
+ *--------------------*/
+
+static lv_font_fmt_txt_glyph_dsc_t glyph_dsc[] =
+{
+    {.bitmap_index = 0,     .adv_w = 23,    .box_h = 13, .box_w = 9, .ofs_x = 2, .ofs_y = 5},
+    {.bitmap_index = 32,    .adv_w = 44,    .box_h = 12, .box_w = 7, .ofs_x = 1, .ofs_y = 3},
+    {.bitmap_index = 55,    .adv_w = 29,    .box_h = 11, .box_w = 8, .ofs_x = 0, .ofs_y = 4},
+};
+
+/*---------------------
+ *  CHARACTER MAPPING
+ *--------------------*/
+
+static uint16_t unicode_list_1 = {0x234, 0x295, 0x456};
+static uint16_t glyph_id_ofs_list_1 = {232, 545, 1466};
+
+static uint8_t glyph_id_ofs_list_3 = {32, 45, 66};
+
+
+/*Collect the unicode lists and glyph_id offsets*/
+static const lv_font_fmt_txt_cmap_t cmaps[] =
+{
+    {
+        .range_start = 0x280,  .range_length = 124,   .type = LV_FONT_FMT_TXT_CMAP_SPARSE_FULL,
+        .glyph_id_start = 223, .unicode_list = unicode_list_1,  .glyph_id_ofs_list = glyph_id_ofs_list_1
```

Comment:
Seems missed "entries count" properties, it's required for sparse tables

Reply:
Let's close the lists with `0`. Can it work?

Reply:
No, because binary search required. It's in spec not for fun :)

Reply:
It's really required for binary search.
For consistency, I added `glyph_id_ofs_list_length` too.
Now both `..._length` is `uint32_t`. If `uint16_t` is also enough I update it.

Reply:
Why not just put `.entries_count` there, as described in spec? https://github.com/littlevgl/lv_font_conv/blob/master/doc/font_spec.md#subtable-header

Renaming & duplicating does not looks like improvement, IMO.

`uint16_t` is enougth (according to spec - 2 bytes)

Reply:
If the two lists always have the same size then fine to have a common `length`.

I updated acordingly.